satellite/satellitedb/statdb.go

```diff
@@ -52,8 +58,12 @@ func (s *statDB) Create(ctx context.Context, nodeID storj.NodeID, startingStats
 		totalUptimeCount   int64
 		uptimeSuccessCount int64
 		uptimeRatio        float64
+		wallet             string
+		email              string
 	)
 
+	fmt.Printf("starting stats: %+v\n", startingStats)
```

Comment:
Don't forget to remove this or transform to debug log if still necessary.

Reply:
Removed 👍